de.fhdo.lemma.service.open_api/.classpath

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8">
+		<attributes>
+			<attribute name="module" value="true"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
+	<classpathentry kind="src" path="src"/>
+	<classpathentry kind="src" path="xtend-gen"/>
+	<classpathentry kind="lib" path="lib/accessors-smart-1.2.jar"/>
```

Comment:
These classpath entries are a bit scary to me (I know why they are here) updatesite-wise. Could you check after compiling the updatesite that the updatesite is installable and the OpenAPI plugin works?